src/components/reusuables/typography/SectionHeader.js

```diff
@@ -0,0 +1,7 @@
+import React from "react"
+
+const SectionHeader = (props) => {
```

Comment:
Change props to the prop name. Example, (props)-> ({heading,subtitle})

Reply:
Props changed to prop name